clustered/server/src/main/java/org/ehcache/clustered/server/internal/messages/EhcacheSyncMessage.java

```diff
@@ -55,19 +55,10 @@ public static SyncOp getSyncOp(byte syncOpCode) {
 
   }
 
-  @Override
-  public Type getType() {
-    return Type.SYNC_OP;
-  }
-
+  @Deprecated
   public abstract SyncOp operation();
```

Comment:
Can't this be removed as well? The only usage is in an error message in `EhcachePassiveEntity` which can be replaced with _getMessageType_ method call.